app/selenium_ui/conftest.py

```diff
@@ -52,7 +52,7 @@ def jsd_dataset(self):
             self.dataset["requests"] = self.__read_input_file(JSD_DATASET_REQUESTS)
             self.dataset["customers"] = self.__read_input_file(JSD_DATASET_CUSTOMERS)
             self.dataset["agents"] = self.__read_input_file(JSD_DATASET_AGENTS)
-            self.dataset["service_desks"] = self.__read_input_file(JSD_DATASET_SERVICE_DESKS)
+            self.dataset["service_desks"] = self.__read_input_file(JSD_DATASET_SERVICE_DESKS_L)
```

Comment:
Do we need JSD_DATASET_SERVICE_DESKS_S here?

Reply:
Fixed